packages/frontend/src/components/molecules/password-strength-indicator.molecule.tsx

```
@@ -0,0 +1,200 @@
+                {meetsMinimum ? '✓ Erfüllt' : `Min. Score: ${minScore}/5`}
+              </Text>
+            )}
+          </div>
+        )}
+      </div>
+
+      {showCriteria && (
+        <div
+          className="space-y-1 rounded-md bg-gray-50 p-3 dark:bg-gray-800"
+          role="list"
+          aria-label="Passwort-Kriterien"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)